src/store/singing.ts

```diff
@@ -1014,32 +1084,72 @@ export const singingStore = createPartialStore<SingingStoreTypes>({
       };
 
       const shiftGuidePitch = (
-        pitchShift: number,
         frameAudioQuery: FrameAudioQuery,
+        pitchShift: number,
       ) => {
         frameAudioQuery.f0 = frameAudioQuery.f0.map((value) => {
           return value * Math.pow(2, pitchShift / 12);
         });
       };
 
-      const scaleGuideVolume = (
-        volumeRangeAdjustment: number,
+      const shiftGuideVolume = (
         frameAudioQuery: FrameAudioQuery,
+        volumeShift: number,
       ) => {
         frameAudioQuery.volume = frameAudioQuery.volume.map((value) => {
-          return value * decibelToLinear(volumeRangeAdjustment);
+          return value * decibelToLinear(volumeShift);
         });
       };
 
-      const calcStartTime = (
-        notes: Note[],
+      const muteLastPauSection = (
```

Comment:
結構文脈の深い処理なので、ちょっとかっこ悪いですが意図を書いておくと新規の方に優しい気がしました！
こんな感じ？
```suggestion
      // 呼吸音が含まれていたりするのでvolumeを制御する
      const muteLastPauSection = (
```

Reply:
コメントを追加しました！